index.php

```diff
@@ -21,6 +21,28 @@
     return $this->renderer->render($response, '/hello.php', $args);
 });
 
+
+$app->get('/login', function ($request, $response, $args) {
+    return $this->renderer->render($response, '/login.php', $args);
+});
+
+$app->post('/login', function ($request, $response, $args) {
+    $body = $request->getParsedBody();
+
+    $userinfo=[];
+    $stmt = $this->database->prepare('select name, password from user where name = ?');
+    $stmt->execute(array($body['login']));
+    while ($row = $stmt->fetch()){
+        $userinfo = $row ;
```

Comment:
Here you are not adding the row to the `$userinfo` array, you are replacing the array.